src/index.html

```diff
@@ -0,0 +1,108 @@
+        <td><input class="f1r1" type="number" min="0" max="10"></td></td>
+        <td><input class="f1r2" type="number" min="0" max="10"></td></td>
+        <th><button id="roll-input-one">Add Scores</button></th>
+        <td id="score">CURRENT SCORE</td>
+      </tr>
+      <tr>
+        <td>2.</td>
+        <td><input type="number" min="0" max="10"></td></td>
+        <td><input type="number" min="0" max="10"></td></td>
+        <th><button id="roll-input-two">Add Scores</button></th>
+        <td id="score">CURRENT SCORE</td>
+      </tr>
+      <tr>
+        <td>3.</td>
+        <td><input type="number" min="0" max="10"></td></td>
+        <td><input type="number" min="0" max="10"></td></td>
+        <th><button id="roll-input-three">Add Scores</button></th>
+        <td id="score">CURRENT SCORE</td>
+      </tr>
+      <tr>
+        <td>4.</td>
+        <td><input type="number" min="0" max="10"></td></td>
+        <td><input type="number" min="0" max="10"></td></td>
+        <th><button id="roll-input-four">Add Scores</button></th>
+        <td id="score">CURRENT SCORE</td>
+      </tr>
+      <tr>
+        <td>5.</td>
+        <td><input type="number" min="0" max="10"></td></td>
+        <td><input type="number" min="0" max="10"></td></td>
+        <th><button id="roll-input-five">Add Scores</button></th>
+        <td id="score">CURRENT SCORE</td>
+      </tr>
+      <tr>
+        <td>6.</td>
+        <td><input type="number" min="0" max="10"></td></td>
+        <td><input type="number" min="0" max="10"></td></td>
+        <th><button id="roll-input-six">Add Scores</button></th>
+        <td id="score">CURRENT SCORE</td>
+      </tr>
+      <tr>
+        <td>7.</td>
+        <td><input type="number" min="0" max="10"></td></td>
+        <td><input type="number" min="0" max="10"></td></td>
+        <th><button id="roll-input-seven">Add Scores</button></th>
+        <td id="score">CURRENT SCORE</td>
+      </tr>
+      <tr>
+        <td>8.</td>
+        <td><input type="number" min="0" max="10"></td></td>
+        <td><input type="number" min="0" max="10"></td></td>
+        <th><button id="roll-input-eight">Add Scores</button></th>
+        <td id="score">CURRENT SCORE</td>
+      </tr>
+      <tr>
+        <td>9.</td>
+        <td><input type="number" min="0" max="10"></td></td>
+        <td><input type="number" min="0" max="10"></td></td>
+        <th><button id="roll-input-nine">Add Scores</button></th>
+        <td id="score">CURRENT SCORE</td>
+      </tr>
+      <tr>
+        <td>10.</td>
+        <td><input type="number" min="0" max="10"></td></td>
+        <td><input type="number" min="0" max="10"></td></td>
+        <th><button id="roll-input-ten">Add Scores</button></th>
+        <td id="score">CURRENT SCORE</td>
+      </tr>
+      </tbody>
+    </table>
+    </div>
+    <div class="buttons">
+      <button id="finish-game">Finish Game</button>
+    </div>
+  </div>
+  
+</body>
+<script src="jsBowling.js"></script>
```

Comment:
Does this indicate that your javascript code is running on the frontend (the browser) or the backend (the server)? 

What Node code does your project have? (Therefore do you need Travis for this project)?